src/utils/Converters.js

```diff
@@ -0,0 +1,125 @@
+// Adapted from https://github.com/geotiffjs/geotiff.js
```

Comment:
We should probably include a copy of the license statement from the geotiffjs project in addition to this link.

Reply:
@abiswas97 I just merged a fix for the converters in PR #11 - since your PR refactors those into this new file I don't think it will merge the changes automatically. Could you grab the updated code for these from the current main branch and incorporate it here?

I also added better attribution for where these came from, including the license statement, which will take care of my comment above.

Reply:
This still needs to be addressed I believe